gammapy/spectrum/spectrum_analysis.py

```diff
@@ -1,12 +1,18 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import absolute_import, division, print_function, unicode_literals
 import logging
+from astropy import wcs
+
 import numpy as np
 from astropy.coordinates import Angle, SkyCoord
 from astropy.extern import six
-from ..image import ExclusionMask
+from astropy.wcs.utils import skycoord_to_pixel
+
+from gammapy.extern import pathlib
```

Comment:
Let's use relative explicit imports everywhere. I.e. here:
```python
from ..extern.pathlib import Path
```
I think this import is superfluous, no?
```python
from gammapy.extern import pathlib
```